Update sso-overview.md (#2918)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/manage-access/sso-overview.md

---

This overview explains how users are provisioned in dbt Cloud via Single Sign-On (SSO).
dbt Cloud supports JIT (Just-in-Time) provisioning and IdP-initiated login. You can learn more about our supported options [here](https://www.getdbt.com/pricing/).

:::tip Configuring SSO 
Once you configure SSO, even partially, for an account, you cannot disable or revert it. When you configure it, you will want to make sure you do so completely.
:::

## Prerequisites

> - You have a dbt Cloud account that is on a dbt Cloud Enterprise plan. If you’re interested in learning more about an Enterprise plan, contact us at sales@getdbt.com.

Reviewer: ```suggestion
- You have a dbt Cloud account that is on a dbt Cloud Enterprise plan. [Contact us](mailto:sales@getdbt.com) if you’re interested in learning more about our Enterprise plan.
```
